recipes/bx/all/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.0)
+project(bx)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+include(GNUInstallDirs)
+
+set(BX_SOURCE_SUBFOLDER "${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder")
+
+file(GLOB BX_SOURCES ${BX_SOURCE_SUBFOLDER}/src/*.cpp)
+list(REMOVE_ITEM BX_SOURCES ${BX_SOURCE_SUBFOLDER}/src/amalgamated.cpp)
+add_library(bx STATIC ${BX_SOURCES})
```

Comment:
Is there any restriction for shared library ?

Reply:
the library does not export the symbols.

Reply:
https://github.com/bkaradzic/bx/blob/74e8c2bd91df850f7142b26a5d85bb3092b606c9/scripts/bx.lua#L7